### PR TITLE
Check laravel requirements (fix https://github.com/laravel/laravel/pull/3625)

### PR DESCRIPTION
**Pull request by aleixfabra**
Check laravel requirements at welcome.blade.php view.

I'm new in laravel so I don't know if you already discussed this in the past but I think it's a great help when you stard a new project.

### CODE REVIEW

**Comment by taylorotwell** [comment]
The requirements should just be documented in the documentation.

File: resources/views/welcome.blade.php [hunk 1]
```diff
@@ -33,12 +33,52 @@
...
     </head>
     <body>
         <div class="container">
             <div class="content">
                 <div class="title">Laravel 5</div>
+
+                @if (version_compare(PHP_VERSION, '5.5.9', '>='))
+                    <p>Your version of PHP is 5.5.9 or higher.</p>
+                @else
+                    <p class="problem">Your version of PHP is too low. You need PHP 5.5.9 or higher to use CakePHP.</p>
```
Comment (GrahamCampbell) [inline resources/views/welcome.blade.php]: > or higher to use CakePHP

:trollface:

**Comment by aleixfabra** [comment]
:+1:

**Comment by AhmetS** [comment]
Forgetting the Cake part... The fact that composer should never allow the install if the wrong PHP version was used makes the version check redundant.

File: resources/views/welcome.blade.php (hunk 1 above)
Comment (Amrit01) [inline resources/views/welcome.blade.php]: :laughing:

**Comment by GrahamCampbell** [comment]
> The fact that composer should never allow the install if the wrong PHP version was used makes the version check redundant.

Not true, you can pass a flag to ignore platform dependencies. Also, there's no reason to assume that this was even installed using composer. People can use the laravel installer.

**Comment by AhmetS** [comment]
I thought the laravel installer used composer? 

Also if you know how to ignore platform dependencies with composer - one would assume that you are also aware that this could cause breaking changes and that you would know how to read the composer.json file which would specify the minimum required PHP version?

**Comment by djallits** [comment]
My humble opinion is this is bloat, respectfully. The requirements are specified in the documentation.

File: resources/views/welcome.blade.php (hunk 1 above)
Comment (jessiesiat) [inline resources/views/welcome.blade.php]: troll  :laughing:

**Pull request closed without merging**